src/plugins/themeAttributes/index.ts

```diff
@@ -32,6 +44,24 @@ export default definePlugin({
                 match: /\.messageListItem(?=,"aria)/,
                 replace: "$&,...$self.getMessageProps(arguments[0])"
             }
+        },
+
+        // add --large-avatar-url css variable to avatar img elements
+        // popout profiles
+        {
+            find: ".LABEL_WITH_ONLINE_STATUS",
+            replacement: {
+                match: /src:null!=\i\?(\i).{1,50}"aria-hidden":!0/,
+                replace: "$&,style:{\"--large-avatar-url\":\"url(\"+$1.replace(/\\d+$/,$self.settings.store.imgSize)+\")\"}"
```

Comment:
try to avoid having too much code inside your replacements, it's hard to maintain and easy to mess up! plus doing it this way allows you to reuse the same func for both avatar patches

```suggestion
                replace: "$&,style:$self.getAvatarStyles($1)"
```